components/SideNavbar/SideNavbarCategory.tsx

```diff
@@ -25,15 +25,14 @@ export const SideNavbarCategory: FC<{
 
   if (isOpen) {
     subcategoryList = (
-      <ul className="relative ml-1">
-        <div className="absolute w-0.5 h-full bg-slate-300 dark:bg-slate-700" />
```

Comment:
Lists (`<ol>`, `<ul>`) must contain only `<li>`, `<script>`, or `<template>` elements. List elements indicate that they contain a list of items, so non-list-item content is semantically weird and confusing.

https://dequeuniversity.com/rules/axe/4.1/list